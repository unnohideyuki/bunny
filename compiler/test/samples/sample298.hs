main = do print $ fmap (*2) (Just 5)
          print $ fmap (*2) Nothing
          print $ fmap (*2) [1, 2, 3]
