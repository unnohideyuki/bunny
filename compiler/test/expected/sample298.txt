Just 10
Nothing
[2,4,6]
